### PR TITLE
move to upstream envoy-openssl and update bazel

### PR DESCRIPTION
**Pull request by mythi**
Depends on envoyproxy/envoy-openssl/pull/10

### CODE REVIEW

File: scripts/build_qat_envoy.sh [hunk 1]
```diff
@@ -197,7 +197,7 @@ build_envoy() {
 	pushd "${ENVOY_DIR}"
 	case $ID in
 		clear-linux*|debian|ubuntu)
-		    CXXFLAGS="-DENVOY_SSL_VERSION=\\\"OpenSSL\\\"" ~/.bazel/bin/bazel build -j "$(jobs)" -c opt //:envoy --define boringssl=disabled
```
Comment (ipuustin) [inline scripts/build_qat_envoy.sh:200]: I think that since envoy-openssl targets the latest Envoy, you'll have to bump bazel version too.

**Comment by mythi** [comment]
@ipuustin @rojkov this is ready now

**Review by rojkov — APPROVED** [review]

**Comment by mythi** [comment]
```
$cat envoy/.bazelversion
1.2.0
$
```

**Comment by ipuustin** [comment]
Looks like the only difference between bazel 1.2.0 and 1.2.1 is a MacOS compatibility change. Since this compiles fine I think it's safe to merge.

**Comment by mythi** [comment]
agree, thanks

**Pull request merged**